src/v2/Apps/Conversation/Components/DetailsSidebar.tsx

```diff
@@ -197,82 +211,164 @@ export const Details: FC<DetailsProps> = ({
                 )}
               </Flex>
             </Flex>
-          </Flex>
+          </StackableBorderBox>
         </>
       )}
-      {!!attachments?.length && (
+      {!!activeOrder && (
         <>
-          <Separator my={2} />
-          <Box px={2}>
-            <Text variant="xs" mb={2}>
+          <TransactionDetailsSummaryItemFragmentContainer
+            order={activeOrder}
+            showOrderNumberHeader
+          />
+          <StackableBorderBox>
+            <Box>
+              <Text variant="md" mb={2}>
+                Ship To
```

Comment:
Question: what should happen to this entire block if the user chooses pick up instead of providing a shipping address? I couldn't find the use case on the figma file.

Also, a small detail based on the design. Would it make sense to have it this way?:
```suggestion
                Ship to
```

Reply:
I spoke with Kseniya to find the use case in the figma file, and this is the one that I think it's missing here:

![image](https://user-images.githubusercontent.com/265560/148221256-7d3667fb-69e0-49c4-a08f-9596be93f727.png)

Reply:
Good catch!
I was about to say it, we use the same idea in Eigen. To double-check you can visit the orders page, which should be like how it is there.